app/assets/javascripts/discourse/app/widgets/header-contents.js

```diff
@@ -5,7 +5,8 @@ createWidget("header-contents", {
   tagName: "div.contents.clearfix",
   transform() {
     return {
-      staff: this.get("currentUser.staff"),
+      showBootstrapMode:
+        this.get("currentUser.staff") && this.get("site.desktopView"),
```

Comment:
I'm pretty sure it's safe to use native getters here 😃 

```suggestion
      showBootstrapMode: this.currentUser?.staff && this.site.desktopView,
```